exercises/concept/ellens-alien-game/.docs/instructions.md

```diff
@@ -0,0 +1,94 @@
+# Instructions
+
+Ellen is making a game where the player has to fight aliens.
+She has just learned about Object Oriented Programming (OOP) and is eager to take advantage of what using `classes` could offer her program.
+
+To Ellen's delight, you have offered to help and she has given you the task of programming the aliens that the player has to fight.
+
+
+## 1. Create the Alien Class
+
+Define the Alien class with a constructor that accepts two `int` parameters `x` and `<y>`, putting them into `x_coordinate` and `y_coordinate` member variables.
```

Comment:
Alien -> \`Alien\` (with backticks)

\`\<y\>\`  -> \`y\` (without the angle brackets)